client/verta/verta/_deployment/endpoint.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import print_function
 
+from verta._internal_utils import _utils
```

Comment:
Relative imports are a bit safer than starting from `verta.`
```suggestion
from .._internal_utils import _utils
```